### ISSUE TITLE
Error: No valueDeclaration for symbol href

### ISSUE THREAD

**Issue by mrlubos** [figma/code-connect#125]
Please provide:

- Code Connect CLI version [use `npx figma -V` if using React, or `figma -V` otherwise, to get the version of your CLI]
v1.0.4

```sh
npx figma connect

┌──────────────────────────────────────────────────────────────────────┐│                                                                      ││                       Welcome to Code Connect                        ││                                                                      ││   Follow a few simple steps to connect your Figma design system to   ││                            your codebase.                            ││     When you're done, you'll be able to see your component code      ││                         while inspecting in                          ││                          Figma's Dev Mode.                           ││                                                                      ││     Learn more at https://www.figma.com/developers/code-connect.     ││                                                                      ││                  Please raise bugs or feedback at                    ││            https://github.com/figma/code-connect/issues.             ││                                                                      ││    Note: This process will create and modify Code Connect files.     ││                          Make sure you've                            ││         committed necessary changes in your codebase first.          ││                                                                      │└──────────────────────────────────────────────────────────────────────┘

No config file found in /Users/mrlubos/path, proceeding with default options
Using "react" parser as package.json containing a "react" dependency was found in /Users/mrlubos/path. If this is incorrect, please check you are running Code Connect from your project root, or add a `parser` key to your config file. See https://github.com/figma/code-connect for more information.
✔ No access token detected. Visit https://help.figma.com/hc/en-us/articles/8085703771159-Manage-personal-access-tokens
      for instructions on how to do this, ensuring you have both the File Content and Code Connect Write scopes 

  Please enter your access token: …


✔ Which top-level directory contains the code to be connected to your Figma design system? (Press enter to use current directory) … 

✔ What is the URL of the Figma file containing your design system library? … https://www.figma.com/design/xxx/

✔ It looks like you don't have a Code Connect config file. Would you like to generate one now from your provided answers? › Yes

Created /Users/mrlubos/path/figma.config.json

┌──────────────────────────────────────────────────────────────────────┐│                                                                      ││                      Connecting your components                      ││                                                                      ││       1 component was automatically matched based on its name        ││                   1 component has not been matched                   ││                                                                      ││     Match up Figma components with their code definitions. When      ││                        you're finished, you                          ││   can specify the directory you want to create Code Connect files    ││                                 in.                                  ││                                                                      │└──────────────────────────────────────────────────────────────────────┘

✖ Select a link to edit (Press esc when you're ready to continue on) › Figma component: button  ↔️ src/components/Form/Button/Button.tsx
✔ What directory should Code Connect files be created in? (Press enter to co-locate your files alongside your component files) … 

✔ You're ready to create 1 Code Connect file. Continue? › Create files

/Users/mrlubos/path/node_modules/@figma/code-connect/dist/react/parser.js:274
            throw new Error(`No valueDeclaration for symbol ${prop.escapedName}`);
                  ^

Error: No valueDeclaration for symbol href
    at extractComponentTypeSignature (/Users/mrlubos/path/node_modules/@figma/code-connect/dist/react/parser.js:274:19)
    at extractSignature (/Users/mrlubos/path/node_modules/@figma/code-connect/dist/connect/wizard/prop_mapping.js:40:74)
    at generatePropMapping (/Users/mrlubos/path/node_modules/@figma/code-connect/dist/connect/wizard/prop_mapping.js:64:23)
    at createCodeConnectFiles (/Users/mrlubos/path/node_modules/@figma/code-connect/dist/connect/wizard/run_wizard.js:355:58)
    at runWizard (/Users/mrlubos/path/node_modules/@figma/code-connect/dist/connect/wizard/run_wizard.js:605:11)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v20.10.0
```

**Comment by ptomas-figma** [comment]
Hi! We'll be fixing this in a new version released today. Thanks for the report!

**Comment by slees-figma** [comment]
Hi @mrlubos, we've now released v1.0.5 which fixes this issue. Sorry for any disruption caused!

**Comment by mrlubos** [comment]
Thanks @ptomas-figma @slees-figma. Are you able to help me figure out https://github.com/figma/code-connect/issues/126? That's the only other blocker I have at the moment
